<?xml version="1.0" encoding="UTF-8"?><testrun name="AppTest" project="secretsharing" tests="9" started="9" failures="1" errors="0" ignored="0">
  <testsuite name="com.shivang.secretsharing.AppTest" time="0.001">
    <testcase name="testA" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testB" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testC" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testD" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testE" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testF" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testG" classname="com.shivang.secretsharing.AppTest" time="0.001">
      <failure>junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at com.shivang.secretsharing.AppTest.testG(AppTest.java:137)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:131)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
    <testcase name="testH" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
    <testcase name="testI" classname="com.shivang.secretsharing.AppTest" time="0.0"/>
  </testsuite>
</testrun>
